src/utils/test-utils.ts

```diff
@@ -23,7 +23,9 @@ export function installMouseEvent() {
   beforeAll(() => {
     const oldMouseEvent = MouseEvent;
     // @ts-ignore
-    global.MouseEvent = class FakeMouseEvent extends MouseEvent {
+    global.MouseEvent = class FakeMouseEvent extends (
+      MouseEvent
+    ) {
```

Comment:
huh, prettier did this? 🤔

Reply:
Yeah seems like the precommit hook is doing this.... i fixed it and commited the code but precommit hook messed it up 
![image](https://user-images.githubusercontent.com/35374649/100056656-0fa81d00-2e4c-11eb-88af-523b24e858f5.png)